.gitignore

```diff
@@ -1 +1,21 @@
+## File system
+.DS_Store
+desktop.ini
+
+## Editor
```

Comment:
fyi there is a gitignore, which is usually used for ignoring project related files, which are shared with everyone
And there is private ignore rules, https://stackoverflow.com/questions/6117109/is-it-possible-to-have-a-custom-gitignore-read-only-access, which are private for each user, and not shared, and my recommendation is to put your ide specific folders into it (because it's hard and stupid to try to support every ide files in gitignore)

So, basically rust inits gitignore with only `target/`, and it should be enough, so you can try to make gitignore as such, and configure exclude to fit you, although it isn't very important and you could leave this file as is